### PR TITLE
Add documentation section to CONTRIBUTING

### PR DESCRIPTION
**Pull request by littlewatkins**
# Pull Request
### Description
I added a section about adding documentation to CONTRIBUTING.md.

This pull request addresses #4

### CODE REVIEW

**Comment by littlewatkins** [comment]
I'll leaving this as a draft until it's reviewed because it could easily be changed due to a number of reasons

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ The naming convention for development branches follows a simple formula:
 where `xx = issue ID number` that the branch is addressing. The brief description
 should only be a two to three words. Each word should be separated by a hyphen.
 
+Documentation
+-------------
+
+Any modules or apps that are created should include documentation, both docstrings
+and comments throughout the code. TurboPy uses the `numpydoc` style for docstrings,
```
Comment (arichar6) [inline CONTRIBUTING.md:31]:
```suggestion
and comments throughout the code. TurboPy uses the [`numpydoc` style](https://numpydoc.readthedocs.io) for docstrings,
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,6 +24,14 @@ The naming convention for development branches follows a simple formula:
...
+and we encourage the same style for those contributing to the project. The
+documentation is then automatically generated using ReadTheDocs.
```
Comment (arichar6) [inline CONTRIBUTING.md:33]:
```suggestion
documentation is then automatically generated and hosted [at ReadTheDocs](https://turbopy.readthedocs.io).
```

**Comment by arichar6** [comment]
oops... guess I should have clicked "request changes"... anyway, you can see my suggestions above.

**Comment by littlewatkins** [comment]
I'll add your changes and then remove the draft status 👍

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -24,6 +24,16 @@ The naming convention for development branches follows a simple formula:
...
 
+Documentation
+-------------
+
+Any modules or apps that are created should include documentation, both docstrings
+and comments throughout the code. TurboPy uses the 
+[`numpydoc` style](https://numpydoc.readthedocs.io) for docstrings,
+and we encourage the same style for those contributing to the project. The
+documentation is then automatically generated and hosted 
+[at ReadTheDocs](https://turbopy.readthedocs.io)..
```
Comment (arichar6) [inline CONTRIBUTING.md:35]:
```suggestion
[at ReadTheDocs](https://turbopy.readthedocs.io).
```

**Pull request merged**